runtime/gateway.go

```diff
@@ -163,6 +166,11 @@ func CreateGateway(
 	if opts.MetricsBackend != nil {
 		metricsBackend = opts.MetricsBackend
 	}
+	if opts.MetricsDefaultBuckets != nil {
+		metricsDefaultBuckets = opts.MetricsDefaultBuckets
+	} else {
+		metricsDefaultBuckets = tally.DefaultBuckets
```

Comment:
Here we are letting the default behavior be rather than the exponential 11 bucket scheme we had set things to previously.  That scheme should be in the specific gateway configuration.